tools/ci/build_libpq.sh

```diff
@@ -116,7 +116,7 @@ if [ "$ID" == "centos" ] || [ "$ID" == "macos" ]; then
         options=(--prefix=${LIBPQ_BUILD_PREFIX} --openssldir=${LIBPQ_BUILD_PREFIX} \
             zlib -fPIC shared)
         if [ -z "${MACOSX_ARCHITECTURE:-}" ]; then
-            ./config $options
+            ./config ${options[*]}
```

Comment:
I assume line 121 is broken too. Can we then do something like (untested, gotta check bash syntax):
```bash
if [ -z "${MACOSX_ARCHITECTURE:-}" ]; then
    options+=(./configure "darwin64-$MACOSX_ARCHITECTURE-cc")
fi
./config ${options[*]}
```
Also isn't `./configure` at line 121 wrong and should be `./config`?